lhotse/audio.py

```diff
@@ -933,7 +933,9 @@ def split(
             )
         ]
 
-    def split_lazy(self, output_dir: Pathlike, chunk_size: int) -> List["RecordingSet"]:
+    def split_lazy(
+        self, output_dir: Pathlike, chunk_size: int, prefix: str
```

Comment:
```suggestion
        self, output_dir: Pathlike, chunk_size: int, prefix: str = ""
```

It seems like prefix could be optional here (and in other functions as well)

Reply:
Yes, it looks good to have a default value. But if `prefix=""`, `(output_dir / prefix)` doesn't work, what about setting default `prefix="split"`?

Reply:
Fixed